qute.ls/com.redhat.qute.ls/src/main/resources/com/redhat/qute/services/snippets/qute-nested-snippets.json

```diff
@@ -1,14 +1,34 @@
 {
-    "#else": {
+    "#case": {
+        "prefix": "case",
+        "body": [
+            "{#case ${1:case}}$0"
+        ],
+        "description": "Case section",
+        "context": {
+            "parent": "switch"
+        }
+    },
+    "#if-else": {
         "prefix": "else",
         "body": [
             "{#else}$0"
         ],
-        "description": "Else section",
+        "description": "Else section for if section",
         "context": {
             "parent": "if"
         }
     },
+    "#for-else": {
+        "prefix": "else",
+        "body": [
+            "{#else}$0"
+        ],
+        "description": "Else section for loop section",
+        "context": {
+            "parent": "for"
```

Comment:
As we can have just one #else inside #for, I suggest that you write context with 

```json
"context": {
            "parent": "for",
            "unique": true
}
```

Reply:
The same applies to `if`, should we add it as well?

Reply:
I dont know I suggest that you test that by writing and parse a qute template